charts/openinwoner/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: openinwoner
 description: Platform voor gemeenten en overheden om producten inzichtelijker en toegankelijker te maken voor inwoners.
 
 type: application
-version: 1.7.0-beta.0
+version: 1.7.0-beta.1
 appVersion: latest
```

Comment:
Do we want to pin this to appVersion 1.27?

Reply:
Yes! And anyway it can be overwritten in the values! It is just a good indication of the minimum value as you say!